packages/core/content-type-builder/admin/src/components/FormModal/index.js

```diff
@@ -510,8 +510,13 @@ const FormModal = () => {
         } else {
           // We cannot switch from collection type to single when the modal is making relations other than oneWay or manyWay
           if (canEditContentType(allDataSchema, modifiedData)) {
+            // allDataSchema.contentType.schema.displayName
             onCloseModal();
 
+            if (modifiedData?.displayName !== allDataSchema?.contentType?.schema?.displayName) {
+              trackUsage('didEditFieldOnContentTypeInCTB');
```

Comment:
just a personal opinion: `didEditFieldOnContentTypeInCTB` doesn't seem clear that it's only the display name of the content type that was changed, it feels like any field of the content type could have been changed, wdyt?

Reply:
yes I think the same, I have asked the same directly to Marco and now in the jira ticket, thanks @ronronscelestes

Reply:
we have changed the event to `didEditFieldNameOnContentType `

Reply:
sounds great :)